config/kubernetes/org.yaml

```diff
@@ -675,6 +675,7 @@ members:
 - ironcladlou
 - irvifa
 - islinwb
+- ithrael
```

Comment:
Note for later: GitHub handles upper cases in usernames very weirdly. We should add usernames in the same case as on GitHub.